src/test/java/com/klarna/hiverunner/ReservedKeywordTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2013 Klarna AB
```

Comment:
I just copied an old test and these copyright javadoc comments are all hidden in Idea :(. But I can change it to 2016 

Should we have these copyright comments at all? What type of OS license do we use?